pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -1756,6 +1756,13 @@ func TestPermitPlugins(t *testing.T) {
 	}
 }
 
+// withMetricsRecorder set metricsRecorder for the scheduling frameworkImpl, only used in tests.
```

Comment:
`only used in tests.` is not needed now.

Reply:
sorry, I cannot get your point here. do you mean the this function is not needed in tests? or update the comment to say this is not needed anymore?

Reply:
Update the comment. :)

`withMetricsRecorder set metricsRecorder for the scheduling frameworkImpl`

Reply:
Done, thanks.